src/QMCWaveFunctions/Fermion/BackflowTransformation.h

```diff
@@ -432,7 +432,6 @@ class BackflowTransformation //: public OrbitalSetTraits<QMCTraits::ValueType>
     activeParticle = iat;
     for (int i = 0; i < NumTargets; i++)
       oldQP[i] = newQP[i] = QP.R[i];
-    const auto& myTable = P.getDistTable(myTableIndex_);
```

Comment:
This looks like it won't compile with AoS.   This statement should be moved inside the #ifndef ENABLE_SOA

Reply:
yep, exactly why CI failed. I didn't check AoS